### PR TITLE
Add header

### PR DESCRIPTION
**Pull request by srikant96**
Screenshot (71)
Screenshot (72)

### CODE REVIEW

File: blogs/01/index.html [hunk 1]
```diff
@@ -5,64 +5,33 @@
...
-                        <li><a href="#">Travel</a></li>
-                        <li><a href="#">Sports</a></li>
-                        <li><a href="#"><img src="img/MagnifyingGlass.svg" alt=""></a></li>
+                        <li><a href="#home">Home</a></li>
+                        <li><a href="#politics">Politics</a></li>
+                        <li><a href="#tech">Tech</a></li>
+                        <li><a href="#entertainment">Entertainment</a></li>
+                        <li><a href="#travel">Travel</a></li>
+                        <li><a href="#sports">Sports</a></li>
+                        <li><a href="#img"><img src="img/search.jpg" alt=""/></a></li>
```
Comment (emphay) [inline blogs/01/index.html:29]: Change reference name not generic one

**Review by emphay — CHANGES_REQUESTED** [review]
MTC

File: blogs/01/styles/common.css [hunk 2]
```diff
@@ -2,7 +2,27 @@ body{
...
+   --primary-font-color: #000;
+   --secondary-font-color: #888;
+   --button-background: #fff;
+   --hover-color:#2f89fc;
+   --primary-background-color:white;
+   --secondary-background-color:#f8f9fa;
+   --hamburger-menu-color:#000;
+   --btn-color-1:#6c757d;
+   --btn-color-2:#8bc34a;
+   --btn-color-3:#f89d13;
```
Comment (amezng) [inline blogs/01/styles/common.css:18]: Use label as the text for the variable

Comment (amezng) [reply blogs/01/styles/common.css:18]: `label-red` `label-yellow` and so on for the variable names

File: blogs/01/styles/header.css [hunk 3]
```diff
@@ -19,91 +18,38 @@ div.slide>ul{
...
-    margin-top: -20px;
 }
 div.slide>ul>li>a{
     text-decoration: none;
     padding: 10px 0px;
-    font-weight: 500;
-    color: #000;
+    font-weight: 620;
+    color: var(--primary-font-color);
     transition: all,0.3s,ease;
```
Comment (amezng) [inline blogs/01/styles/header.css:27]: `transition: all .3s ease-in-out` is the correct value

**Review by amezng — CHANGES_REQUESTED** [review]
MTC

**Review by amezng — APPROVED** [review]

**Comment by amezng** [comment]
closing as this section was merged by @emphay

**Pull request closed without merging**